database.go

```diff
@@ -332,7 +369,7 @@ func (m *Measurement) idsForExpr(n *influxql.BinaryExpr) (seriesIDs, bool, influ
 	}
 
 	// ignore time literals
-	if _, ok := value.(*influxql.TimeLiteral); ok {
+	if _, ok := value.(*influxql.TimeLiteral); ok || name.Val == "time" {
```

Comment:
If they do a `WHERE time > 1s` then it ends up going to this. This is just during the planning phase of the query so a string compare isn't really that big of a deal.